iphone/Maps/UI/EditBookmark/BookmarkUIUtils.swift

```diff
@@ -1,6 +1,9 @@
 import Foundation
 
-extension BookmarkColor {
+extension BookmarkColor: CaseIterable {
+  public static var allCases: [BookmarkColor] =  [.red, .pink, .purple, .deepPurple, .blue, .lightBlue, .cyan, .teal, .green,
```

Comment:
Isn't it already defined in `BookmarkColorViewController`? Let's avoid copy-paste.

Reply:
I replaced it in `BookmarkColorViewController` with:
`private let colors: [BookmarkColor] = BookmarkColor.allCases`